cluster/gce/gci/configure-helper.sh

```diff
@@ -1644,6 +1644,9 @@ function start-node-problem-detector {
     local -r sm_config="${KUBE_HOME}/node-problem-detector/config/systemd-monitor.json"
     local -r ssm_config="${KUBE_HOME}/node-problem-detector/config/system-stats-monitor.json"
 
+    # replace the known-modules.json file path in system-stats-monitor.json with the absolute path
+    sed -i "s|config/guestosconfig/known-modules.json|${KUBE_HOME}/node-problem-detector/config/guestosconfig/known-modules.json|g" "${KUBE_HOME}/node-problem-detector/config/system-stats-monitor.json"
```

Comment:
This seems fragile. How is this tested?

/cc @SergeyKanzhelev

Reply:
I agree, this is a bit fragile. 

The current issue we're seeing from NPD is:
```
May 04 21:44:43 gke-cs-1-20-prod-default-pool-bca458cf-r1f5 node-problem-detector[1653]: W0504 21:44:43.979805    1653 osfeature_collector.go:102] Failed to read configuration file config/guestosconfig/known-modules.json: open config/guestosconfig/known-modules.json: no such file or directory
```
That seems to be coming from here: https://github.com/kubernetes/node-problem-detector/blob/master/pkg/systemstatsmonitor/osfeature_collector.go#L100

I think the issue is that NPD is running from `/home/kubernetes/bin`, but the configs are all placed in `/home/node-problem-detector/config`. 

Trying to read that path (`config/guestosconfig/known-modules.json:)` `https://github.com/kubernetes/node-problem-detector/blob/031e658/config/system-stats-monitor.json#L95` using relative path doesn't work since NPD binary and config are in different paths.

It would be ideal if NPD could support an explicit flag for `known-modules.json` or maybe a directory where all the configs should be placed, but sounds like it will require NPD changes. @vteratipally do you have any thoughts?

Reply:
Yes I agree, I was planning to add a path to command line, for the known-modules.json instead of configuring it from the system-stats-monitor.json. But until then to mitigate the error message till we upgrade NPD, I thought this PR should help.

Reply:
> It would be ideal if NPD could support an explicit flag for known-modules.json or maybe a directory where all the configs should be placed, but sounds like it will require NPD changes... @vteratipally do you have any thoughts?

I did take a look and the way npd is designed it would be hard to configure a known_modules path to the command line. And technically npd/config is the place where we place all the configs.